book/src/commands/find.md

```diff
@@ -169,7 +169,12 @@ When provided, the trailing command (`cmd`) will be used as the _cargo-msrv chec
 
 # EXAMPLES
 
-1. Try to determine the MSRV for the crate in your current working directory, using the binary search strategy.
+1. Determine the MSRV for the crate in your current working directory, while ignoring `rust-version` fields in crates.
```

Comment:
Please change the order, so this new example will be the 4th example. 

The 1st example should be the default optionless `cargo msrv find` and I like the 2nd to be `cargo msrv find --linear`, to keep similar options in order, next to each other.

The 3rd example introduced the concept "a custom cargo-msrv check command".

Adding it as the 4th example would then make the most sense, since the prior example introduced the concepts needed to understand this one.

Reply:
Perhaps it would also be good to note here that the `--ignore-rust-version` option was introduced in cargo 1.56, so this will also only test rust versions >= 1.56